assets_management/wizard/account_move_manage_asset_view.xml

```diff
@@ -151,7 +150,6 @@
                                 <field
                                     name="company_id"
                                     options="{'no_open':1, 'no_create_edit': True}"
-                                    invisible="1"
                                     readonly="1"
```

Comment:
Qui è stato lasciato il readonly e tolto l'invisible, è voluto?

Reply:
Probabilmente era per testarlo perchè lasciandolo `invisibile` non avrei avuto la prova immediata.
Per le move lines sono tutti campi visibili, ma readonly, per me è uguale seguire questa linea anche per l'azienda o lasciarlo invisibile, ditemi voi cosa è meglio.

Reply:
L'avevo evidenziato perché era l'unica modifica diversa dalle altre https://github.com/OCA/l10n-italy/compare/78caca0394e0c74f4a86946d032019fd106da10f..e9d0a5e216d9ff7015543fe9561e9b2b61f3753d

Reply:
alla fine mi sono reso conto che sia `currency_id` che `company_id` sono già in sola lettura da modello perciò l'ho tolto da XML ed ho lasciato `invisible`
cosa ne pensi?

Reply:
ok